.github/workflows/releases.yml

```diff
@@ -0,0 +1,37 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Update Releases
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push events but only for the master branch
+  push:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  update-tables:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-20.04
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a set of commands using the runners shell
+      - name: Generate Translation Status Tables
+        run: |
+          sudo apt-get update -y >/dev/null 2>&1
+          sudo apt-get install -y python3 gettext >/dev/null 2>&1
+          cd .applet-versions
+          python3 release_generator.py
+          git config user.name "NikoKrause"
+          git config user.email "8415124+NikoKrause@users.noreply.github.com"
```

Comment:
Should this have one user's credentials hard-coded?

Reply:
It's been a long time, but I think I copied it from the workflow to generate the translation tables.

I'm not sure what I should be using, if it were on me I would use a generic CI account

Reply:
I'm going to take a shot at adding some workflows in a few weeks that will work on a similar basis. Let's see what we can come up with that is a cleaner design.